cli/cmd/deploy/deploy.go

```diff
@@ -277,6 +283,7 @@ func DeployCmd(cfg *config.Config) *cobra.Command {
 
 	deployCmd.Flags().SortFlags = false
 	deployCmd.Flags().StringVar(&projectPath, "path", ".", "Project directory")
+	deployCmd.Flags().StringVar(&subPath, "sub-path", "", "Project path to sub directory of a larger repository")
```

Comment:
1. Change description to `Relative path to project in the repository (for monorepos)`
2. Move it to before the `prod-branch` flag, it will be a rare flag, so no need to have it early in the list